.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: go
+
+go:
+  - 1.11.x
+
+go_import_path: gopkg.in/src-d/go-billy-gluster.v0
+
+dist: xenial
+addons:
+  apt:
+    update: true
+    packages:
+      - glusterfs-common
+
+matrix:
```

Comment:
Given that we don't use `tip`, you can drop this `matrix` section altogether.

Reply:
Done